Week3/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>JS-week3</title>
+</head>
+<body 
+style ="text-align: center; padding: 10vw;">
+
+    <h1 style="color: red; font: 3em sans-serif;">JS-week3</h1>
+    
+    <script src="strings.js"></script>`
```

Comment:
I think you forgot some parts of the homework?

I think assignment 12 untill 16 are missing, see the [homework](https://github.com/HackYourFuture/JavaScript1/blob/master/Week3/MAKEME.md)